i18n/nl.yaml

```diff
@@ -21,6 +21,9 @@ article:
         other: Gerelateerde inhoud
     lastUpdatedOn:
         other: Laatst bijgewerkt op
+    readingTime:
+        one: "{{ .Count }} leestijd"
+        other: "{{ .Count }} leestijd"
```

Comment:
Hi, sorry for the late reply and thanks for the PR!

Those two translations are the same right? If that's the case, you can remove `one: "{{ .Count }} leestijd"` and it will fallback to `other: "{{ .Count }} leestijd"`